Overview.src.html

```diff
@@ -4841,6 +4955,10 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
 getter must return <span title=concept-Request-request>request</span>'s
 <span title=concept-request-credentials-mode>credentials mode</span>.
 
+<p>The <dfn title=dom-Request-use-referred-token-binding><code>useReferredTokenBinding</code></dfn>
+attribute's getter must return <span title=concept-Request-request>request</span>'s
+<span title=concept-request-use-referred-token-binding>use-referred-token-binding flag</span>.
```

Comment:
A flag is either set or unset and this wants to return a boolean. So unfortunately you'll have to do a conditional.